daemon/container_operations_unix.go

```diff
@@ -718,6 +722,11 @@ func (daemon *Daemon) DisconnectFromNetwork(container *container.Container, n li
 	if err := container.ToDiskLocking(); err != nil {
 		return fmt.Errorf("Error saving container to disk: %v", err)
 	}
+
+	attributes := map[string]string{
+		"container": container.ID,
+	}
+	daemon.LogNetworkEventWithAttributes(n, "disconnect", attributes)
 	return nil
 }
 
```

Comment:
yeah, the fact that the caller is in `daemon/start.go` always confuses me. Thank you.